privileged-session.md

```diff
@@ -0,0 +1,78 @@
+### Establishing a privileged session
+The following interactions happen between cloud.gov systems when the data.gov team establishes an authenticated SSH session. See the Cloud Foundry documentation for additional details if necessary (https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-apps.html#other-ssh-access).
+
```

Comment:
Can you add the preview image here?

Reply:
![image](https://user-images.githubusercontent.com/75858167/103218802-18c45800-48ea-11eb-8b36-50f41dfde3b4.png)